.github/workflows/docker_build_and_test.yml

```diff
@@ -32,9 +32,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Set up Python 3.10
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
```

Comment:
It seems the latest version is `v5`, so should we try it?

Reply:
Sure